src/main/java/net/dv8tion/jda/internal/entities/ReceivedMessage.java

```diff
@@ -1087,6 +1087,24 @@ public void setMentions(List<User> users, List<Member> members)
         return collection;
     }
 
+    private IPermissionContainer getPermissionContainer()
+    {
+        IPermissionContainer permChan;
+
+        // Check if the channel is a thread
+        if (getChannelType().isThread())
+        {
+            // Get the parent channel for permissions if we are using a thread
+            permChan = ((ThreadChannel) channel).getParentChannel();
+        }
+        else
+        {
+            permChan = (IPermissionContainer) channel;
+        }
+
+        return permChan;
+    }
```

Comment:
This is entirely unneeded. `GuildChannel#getPermissionContainer() : IPermissionContainer` exists and should be used when needing to get an `IPermissionContainer` in unknown concrete channel situations. 

However, we don't actually need an `IPermissionContainer` as stated above.